components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -2965,11 +2965,72 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /settings/scopes/{scope}:
+    #-----------------------------------------------------
+    # Retrieve scope settings
+    #-----------------------------------------------------
+    get:
+      security:
+        - OAuth2Security:
+            - apim:admin_settings
+      x-wso2-curl: "curl -k -X GET \"https://localhost:9443/api/am/admin/v1/settings/scopes/apim:subscribe?username=john\" -H \"accept: application/json\" -H \"Authorization: Bearer 8e8edc50-4a70-3fec-af00-ce08097012ab"
+      x-wso2-request: "GET https://localhost:9443/api/am/admin/v1/settings/scopes/apim:subscribe?username=john Authorization: Bearer 8e8edc50-4a70-3fec-af00-ce08097012ab"
```

Comment:
let's have a habit to mention the x-wso2-request in this format in future PRs. https://github.com/Meruja/carbon-apimgt/blob/5510d27701692c0e352ff2194d6a95d43c49d3ea/components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml#L2301-L2303.  It is more clear.